app/assets/markdown/getting-started-zh-HANS.md

```diff
@@ -0,0 +1,132 @@
+##### 开启旅程
+# 10分钟上手CodeCombat教学版！
+#### 让您的班级通过下列步骤开启编程的奇妙旅程：
+
+##### 第1步
+## 创建教师帐号
+
+创建教师帐号后，您将能够创建班级、邀请学生、监控学生进度、添加学生、在添加学生后为学生分配课程内容。
+
+在账号创建时的选择帐号类型界面，选择**“我是老师”**选项，注册成为教师。
+
+<img src="https://assets.koudashijie.com/images/image1.png" title="创建教师帐号" alt="创建教师帐号" width="400px" />
```

Comment:
I think we should put these files in our GitHub repo, along with the English versions of them, instead of on assets.koudashijie.com and files.codecombat.com. Otherwise, people looking at the English version within China or the Chinese version outside of China will still get the wrong domains. This would also make it easier to keep resources in sync.

I recommend /app/assets/images/pages/teachers/resources/markdown/ as this folder, duplicating existing assets there, and updating these resource hub markdown files to point there.